README.md

```diff
@@ -46,8 +46,6 @@ The quickest way to start up Service Telemetry Framework for development is to
 run the `quickstart.sh` script located in the `deploy/` directory after starting
 up a [CodeReady Containers](https://github.com/code-ready/crc) environment.
 
-To deploy a local build of the Service Telemetry Operator itself, start by
```

Comment:
:-1: Did something go wrong here? This doesn't look right.

Reply:
Oh, I think I must have started to update this section to account for the new CI. Let me review and adjust.